lib/obj/Cargo.toml

```diff
@@ -14,5 +14,7 @@ edition = "2018"
 [dependencies]
 cranelift-codegen = "0.28.0"
 cranelift-entity = "0.28.0"
+cranelift-wasm = "0.28.0"
+wasmtime-runtime = { path = "../runtime" }
```

Comment:
As a general principle, we should avoid having wasmtime-obj depend on wasmtime-runtime, because wasmtime-runtime assumes it's running on the target, while wasmtime-obj could be used for cross compilation. I'll add comments below for how to avoid these dependencies.